Documentation/v2/etcd2.rules

```diff
@@ -0,0 +1,121 @@
+### General cluster availability ###
```

Comment:
change the file name to etcd_alert.rules?

this is under v2 dir, no need to say etcd2.